### PR TITLE
Add Deployment URL

### PR DESCRIPTION
**Pull request by denysvitali**
This PR is tightly related to https://github.com/hashicorp/waypoint/pull/1387 and https://github.com/hashicorp/waypoint-plugin-sdk/pull/27.

Thanks to this PR, the deployment URL is now displayed in the CLI when the process is done.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -31,7 +31,7 @@ require (
 	github.com/tedsuo/rata v1.0.0 // indirect
 	github.com/vito/go-interact v1.0.0 // indirect
 	google.golang.org/protobuf v1.26.0
-	k8s.io/apimachinery v0.19.4 // indirect
+	k8s.io/apimachinery v0.19.4
 )
 
-// replace github.com/hashicorp/waypoint-plugin-sdk => ../../waypoint-plugin-sdk
+replace github.com/hashicorp/waypoint-plugin-sdk => github.com/swisscom/waypoint-plugin-sdk v0.0.0-20210426082641-d2f8c888d724
```
Comment (LarsBaertschi) [inline go.mod:37]: this should be temporary and we should remove it again once the change on the SDK is merged. But I'm fine with merging it.

File: platform/deploy.go [hunk 2]
```diff
@@ -90,43 +91,40 @@ func (p *Platform) DeployFunc() interface{} {
...
 //
 // The output parameters for BuildFunc must be a Struct which can
-// be serialzied to Protocol Buffers binary format and an error.
+// be serialized to Protocol Buffers binary format and an error.
 // This Output Value will be made available for other functions
 // as an input parameter.
 // If an error is returned, Waypoint stops the execution flow and
 // returns an error to the user.
-func (b *Platform) deploy(ctx context.Context, log hclog.Logger, ui terminal.UI, img *docker.Image, job *component.JobInfo, source *component.Source, deploymentConfig *component.DeploymentConfig) (*Deployment, error) {
+func (p *Platform) deploy(ctx context.Context, log hclog.Logger, ui terminal.UI, img *docker.Image, job *component.JobInfo, source *component.Source, deploymentConfig *component.DeploymentConfig) (*Deployment, error) {
```
Comment (LarsBaertschi) [inline platform/deploy.go:99]: no clue why I ever named this "b", thanks!

**Review by LarsBaertschi — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (denysvitali) [reply go.mod:37]: Actually, I don't mind waiting for a bit until the SDK is updated too. I'll keep this PR open until the SDK PR is merged 🚀

**Comment by denysvitali** [comment]
Merging, because I need to add more stuff and I rely on this PR 👍

**Pull request merged**